### PR TITLE
Developers sub page

### LINKED ISSUE #361: Developers sub-page
We embrace community PRs and PRs made through gitcoin support.

as we plan to be community owned project we would like to have more landing area for new developers

### PR DESCRIPTION
**Pull request by hisergiorojas**
**Thank you for your contribution** to the KodaDot NFT gallery.

👇 Let's do a quick check before the merge. 

### PR type
- [ ] Bugfix 
- [x] Feature 
- [ ] Refactoring 

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Optional
- [ ] I've tested PR on mobile and everything works
- [ ] I found edge cases

### What's new? 
- [x] PR closes #361 
- [ ]  By visiting http://localhost:9090/developers 
I added a developer page because we can have a page for new developers that want to contribute to the project.

### Had issue bounty label ?
- [ ] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=<My_Kusama_Address>)

### Community participation
- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [x] My fix has changed **something** on UI, a screenshot for others, is best to understand changes.

### CODE REVIEW

**Comment by hisergiorojas** [comment]
Screenshot from 2021-11-09 11-51-58

**Comment by yangwao** [comment]
Before merging this we will merge `main` into `main-nuxt` to sync the stuff as to be sure everything is all right https://github.com/kodadot/nft-gallery/issues/1077

File: src/components/landing/Developers.vue [hunk 1]
```diff
@@ -0,0 +1,302 @@
+<template>
+  <div>
+    <section class="hero is-dark">
+      <div class="hero-body">
+        <div class="container">
+          <div class="columns">
+            <div class="column is-6 is-offset-3">
```
Comment (roiLeo) [inline src/components/landing/Developers.vue:7]: I would have remove these ```<div>``` to match container size (as seen in other pages)

File: src/components/landing/Developers.vue [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+              <h1 class="title is-1">
+                Developers
+              </h1>
+              <h1 class="subtitle">
```
Comment (roiLeo) [inline src/components/landing/Developers.vue:11]: Bad pratice (SEO) to use multiple h1, use h2 for subtitle

**Review by roiLeo — CHANGES_REQUESTED** [review]
branch outdated?
- You need to change other `<h1>` tag to something else
- change container size to match other pages
- not a big fan of `<br>`

**Comment by yangwao** [comment]
> branch outdated?
> 
> * You need to change other `<h1>` tag to something else
> * change container size to match other pages
> * not a big fan of `<br>`

Hey @hisergiorojas  Let's fix conflicts, avoid br and use h1,h2, accordingly to SEO practices

**Comment by hisergiorojas** [comment]
Okay, I will fix the issue.

**Comment by yangwao** [comment]
How users will find about this? Probably put link on the landing page. 

This should be emoji 'sparkles' ✨
image

> The good first issue **label isfor problems** or updates we think are ideal for beginners. 

Probably space is missing
image

Checking against recent [contributing.md](https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md)

Seems you are missing whole [section about Rewards](https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#rewards)

Let's add it please. 
image

**Comment by yangwao** [comment]
Added [Dynamics of PRs and avoiding stalled PRs section](https://github.com/kodadot/nft-gallery/blob/main-nuxt/CONTRIBUTING.md#dynamics-of-pull-requests)

**Comment by yangwao** [comment]
Converted to draft as it's not ready for merge.
Lmk when you reflect new changes @hisergiorojas

**Comment by yangwao** [comment]
Hey, I'm closing this as seems to lack progress and attention from PR author.

**Pull request closed without merging**